### ISSUE TITLE
[Question]  loss 不能下降

### ISSUE THREAD

**Issue by awzhgw** [haotian-liu/LLaVA#1375]
### Question

When I fine-tune with Lora, the loss will drop to about 0.8 in the first 20k steps, and then increase to 6.2 over time, and then keep oscillating here. May I ask why this problem occurs? How to solve it? Translate this passage into English

**Comment by Sprinter1999** [comment]
Hi friend, I guess you may consider switching a lower learning rate when conducting your SFT.

**Comment by OliverLeeXZ** [comment]
Same question

**Comment by Vignesh-Valaboju** [comment]
Same problem too!

**Comment by OliverLeeXZ** [comment]
I change learning rate from 2e-4 to 2e-5. It is really work!
